repo.go

```diff
@@ -671,6 +672,38 @@ func (r *Repo) AddTargets(paths []string, custom json.RawMessage) error {
 	return r.AddTargetsWithExpires(paths, custom, data.DefaultExpires("targets"))
 }
 
+func (r *Repo) AddDigestTargets(digest string, length int64, custom json.RawMessage) error {
+	expires := data.DefaultExpires("targets")
+
+	t, err := r.targets()
+	if err != nil {
+		return err
+	}
+	hashes := make([]string, 1)
```

Comment:
Since we never actually use `hashes` as slice, only access its zeroth element and its length, could we just use a `hash` `string` variable?

Reply:
Yes, I was originally thinking that this could support multiple hashes, but the rest of the code doesn't yet, so I'll make this a string.